app/mailers/super_admin_mailer.rb

```diff
@@ -2,10 +2,19 @@ class SuperAdminMailer < ApplicationMailer
   default from: 'remote-flow@mail.com'
 
   def new_organization_email(organization)
-    admins_email = User.super_admins.pluck(:email)
-    return false if admins_email.empty?
-
     @organization = organization
-    mail(to: admins_email, subject: 'New organization created')
+    mail(to: super_admin_emails, subject: 'New organization created') unless super_admin_emails.empty?
+  end
+
+  def send_statistic_about_new_users_and_organizations
+    @users_count = User.where(["DATE(created_at) >= ?", Date.today - 7.days]).count
+    @organizatinos_count = Organization.where(["DATE(created_at) >= ?", Date.today - 7.days]).count
+    mail(to: super_admin_emails, subject: 'Weekly statistics') unless super_admin_emails.empty?
```

Comment:
note: when there're no super admin emails, then  there's no reason to query those counts above